lib/Dist/Zilla/Plugin/Alien.pm

```diff
@@ -505,19 +506,30 @@ use strict;
 use warnings;
 use File::ShareDir ':ALL';
 use Path::Class;
-
-my $abs = file(dist_dir('{{ $dist->name }}'),'bin','{{ $bin }}')->cleanup->absolute;
+use File::Which;
```

Comment:
We should add File::Which as a prerequisite when using this option, if not already added.  (gets pulled in already by AB, but we shouldn’t rely on that here).

Reply:
Ah, that needs to go away. I removed the code in the else-branch that used `which()`.